app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    before_action :configure_sign_up_params, only: [:create]
+    # before_action :configure_account_update_params, only: [:update]
```

Comment:
I'd remove all I don't need

Reply:
I'll be honest, I decided to leave all the commented code because this is something new for me and I'm not pretty sure if I'll need something later (guess I'll do). 

Can I leave it for now and remove it later?